packages/wallet/frontend/src/lib/api/paymentPointer.ts

```diff
@@ -40,6 +40,11 @@ export type PaymentPointer = {
   keyIds: PaymentPointerKey | null
 }
 
+export type ListPaymentPointersResult = {
+  wmPaymentPointers: Array<PaymentPointer>
+  paymentPointers: Array<PaymentPointer>
+}
```

Comment:
`listAll` function needs a separate type for the response, since it's not returning `{ paymentPointers: [], wmPaymentPointers: [] }`.